Modules/Order/Http/Services/OrderService.php

```diff
@@ -208,97 +220,37 @@ public function checkout($request): array
-                    if (!empty($prdctArr)) {
-                        foreach ($prdctArr as $prdctk => $prdctv) {
-                            $product = Products::where('id', $prdctv['product_id'])->first();
-                            $cartArr[$brandk]['products'][$prdctk]['id'] = $prdctv['id'];
-                            $cartArr[$brandk]['products'][$prdctk]['product_id'] = $product->id;
-                            $cartArr[$brandk]['products'][$prdctk]['product_name'] = $product->name;
-                            $cartArr[$brandk]['products'][$prdctk]['product_price'] = (float)$prdctv['price'];
-                            $cartArr[$brandk]['products'][$prdctk]['product_qty'] = (int)$prdctv['quantity'];
-                            $cartArr[$brandk]['products'][$prdctk]['product_image'] = $product->featured_image != '' ? $product->featured_image : asset('public/admin/dist/img/logo-image.png');
-                        }
-                    }
-                }
-            }
-            if ($order->country) {
-                $country = DB::table('countries')->where('id', $order->country)->first();
-                $order->country = $country->name;
-            }
-            if ($order->state) {
-                $state = DB::table('states')->where('id', $order->state)->first();
-                $order->state = $state->name;
-            }
-            if ($order->town) {
-                $town = DB::table('cities')->where('id', $order->town)->first();
-                $order->town = $town->name;
-            }
-            $oprdctArr = [];
-            $ocartArr = Cart::where('user_id', $request->user_id)->where('order_id', $order->id)->where('brand_id', $brandv['brand_id'])->get()->toArray();
-            $ototalPrice = 0;
-            $ototalQty = 0;
-            if (!empty($ocartArr)) {
-                foreach ($ocartArr as $prdctk => $prdctv) {
-                    $sub_total = (float)$prdctv['price'] * (int)$prdctv['quantity'];
-                    $ototalQty += (int)$prdctv['quantity'];
-                    $ototalPrice += $sub_total;
-                    $oprdctArr[] = array(
-                        "sku" => $prdctv['product_sku'],
-                        "name" => $prdctv['product_name'],
-                        "desc" => $prdctv['style_name'],
-                        "qty" => (int)$prdctv['quantity'],
-                        "price" => (float)$prdctv['price'],
-                        "subtotal" => $sub_total,
-                    );
-                }
-            }
-            $data = array(
-                'order_det' => $order,
-                'cart_det' => $cartArr,
-            );
-            $response = ['res' => true, 'msg' => 'Your product successfully placed in order', 'data' => $data];
+            $this->syncExternal($prdArr, $brand->user_id);
+
+            $response = ['res' => true, 'msg' => 'Your product successfully placed in order', 'data' => ''];
         }
         return $response;
```

Comment:
Add line above and remove empty line below.

Reply:
Done